redesign/css/dwyl.css

```diff
@@ -60,6 +60,14 @@ body {
   font-size: 4rem;
 }
 
+.bw-s {
+  border-width: thin;
```

Comment:
I haven't used this before ... 
https://developer.mozilla.org/en/docs/Web/CSS/border-width#Syntax
learned something new today ...
thanks Cleo. 😉